SEO excerpts: Use part 4 (#3838)
timescale/docs: use-timescale/extensions/pgvector.md

---
title: pgvector PostgreSQL extension

> excerpt: pgvector is an open-source extension for PostgreSQL that enables efficient vector similarity search and can be used with Timescale CLoud. Learn to use pgvector to create a chatbot

Reviewer: ```suggestion
excerpt: pgvector is an open-source extension for PostgreSQL that enables efficient vector similarity search that you can enable in Timescale Cloud. Learn to use pgvector to create a chatbot
```
